src/core/configs/configRoutes.js

```diff
@@ -0,0 +1,30 @@
+const ROUTES_HOME = {
+  HOME: { NAME: 'Home', PATH: '' },
+  //A-Z
+  ACCESS_WALLET: { NAME: 'AccessWallet', PATH: 'wallet/access/:overlay?' },
+  BUY_HARDWARE_WALLET: { NAME: 'BuyHardwareWallet', PATH: 'company' },
+  COMPANY_PAGE: { NAME: 'CompanyPage', PATH: 'company' },
+  CREATE_WALLET: { NAME: 'CreateWallet', PATH: 'wallet/create/:overlay?' },
+  HOW_IT_WORKS: { NAME: 'HowItWorks', PATH: 'how-it-works' },
+  PRESS_KIT: { NAME: 'PressKit', PATH: 'presskit' },
+  PRIVACY_POLICY: { NAME: 'PrivacyPolicy', PATH: 'privacy-policy' },
+  SECURITY_POLICY: { NAME: 'SecurityPolicy', PATH: 'security-policy' },
+  TEAM_PAGE: { NAME: 'TeamPage', PATH: 'team' },
+  TERMS_OF_SERVICE: { NAME: 'TermsOfService', PATH: 'terms-of-service' },
+  TOOLS: { NAME: 'Tools', PATH: 'tools' }
+};
+const ROUTES_WALLET = {
+  WALLETS: { NAME: 'Wallets', PATH: '' },
+  //A-Z
+  DAPPS: { NAME: 'Dapps', PATH: 'dapps' },
+  DASHBOARD: { NAME: 'Dashboard', PATH: '/dashboard' },
```

Comment:
why does this one has a `/` on PATH

Reply:
It was existing in the path from routes-default, but doesn't like it's needed- i'll go ahead and remove it